### PR TITLE
sci-libs/spooles: resurrect

### PR DESCRIPTION
**Pull request by khumarahn**
Hi, this is the resurrection ritual for spooles. I updated the old ebuild to EAPI 7 but did not do much otherwise.

As discussed in https://github.com/waebbl/waebbl-gentoo/issues/246.

### CODE REVIEW

File: sci-libs/spooles/spooles-2.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,72 @@
+# Copyright 1999-2012 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit eutils toolchain-funcs flag-o-matic
```
Comment (waebbl) [inline sci-libs/spooles/spooles-2.2.ebuild:5]: Is eutils needed? AFAIK it's rarely needed with EAPI 7

**Review by waebbl — CHANGES_REQUESTED** [review]

File: sci-libs/spooles/spooles-2.2.ebuild (hunk 1 above)
Comment (khumarahn) [reply sci-libs/spooles/spooles-2.2.ebuild]: You are right, it seems that eutils are not used. At least it compiles fine without it.

I made the two changes and force-pushed.

File: sci-libs/spooles/spooles-2.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,72 @@
+# Copyright 1999-2012 Gentoo Foundation
```
[the remark this answers is not in the document]
Comment (waebbl) [reply sci-libs/spooles/spooles-2.2.ebuild]: It needs to read Gentoo Authors. I'm sorry, `old header format` was probably a bit vague.

[the remark this answers is not in the document]
Comment (khumarahn) [reply sci-libs/spooles/spooles-2.2.ebuild]: changed!

**Review by waebbl — APPROVED** [review]
Looks good and builds on my machine.
Thanks for your contribution!

**Comment by khumarahn** [comment]
very good, thanks a lot!

**Pull request merged**